### PR TITLE
Add note on DDSQL Editor permissions

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add a note on DDSQL Editor RBAC permissions
- [DOCS-9354](https://datadoghq.atlassian.net/browse/DOCS-9354)

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### Additional notes

[DOCS-9354]: https://datadoghq.atlassian.net/browse/DOCS-9354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by wbew — COMMENTED** [review]
This looks great! 

The team is having some new discussions around the permission and roles that might affect the documentation -- I'll hold off on approving for now, apologies for the shifting ground. Hope to have an update next week. 

Thanks again!

**Review by drichards-87 — APPROVED** [review]

File: content/en/ddsql_editor/_index.md [hunk 1]
```diff
@@ -68,9 +68,14 @@ Browse and re-run saved queries in the side panel.
 
 {{< img src="ddsql_editor/saved-queries-panel.png" alt="A list of saved queries" style="width:60%;" >}}
 
+## Permissions
+
+To access the DDSQL Editor app, users need the `ddsql_editor_read` permission. This permission is included in the Datadog Standard Role by default. If your organization uses custom roles, add this permission to the appropriate role. For more information on managing permissions, see the [RBAC documentation][3].
```
Comment (wbew) [inline content/en/ddsql_editor/_index.md:73]:
```suggestion
To access the DDSQL Editor app, users need the `ddsql_editor_read` permission. This permission is included in the Datadog Read Only Role by default. If your organization uses custom roles, add this permission to the appropriate role. For more information on managing permissions, see the [RBAC documentation][3].
```

Hey @estherk15! Circled back with the team, and we've decided to place it into the `Read Only` role instead.

We'll have to make some code changes that may take a while. I will stamp the approval when the doc change is ready to go live. Thanks again!

Comment (estherk15) [reply content/en/ddsql_editor/_index.md:73]: Sounds good @wbew, I'll put a Do Not Merge label. When this is good to go, feel free to remove and comment `/merge`!

**Review by wbew — APPROVED** [review]

**Pull request merged**